draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -756,17 +748,15 @@ To this end, Verifiers MUST follow the following steps (or equivalent):
    details.
 2. Process the SD-JWT as defined in (#sd_jwt_verification).
 3. If Key Binding is required:
```

Comment:
```suggestion
3. If Key Binding JWT is required:
```

if Key Binding is required, but can be provided using means other than KB JWT, the current 3.1 is too strict